docs/releases/1.19-NOTES.md

```diff
@@ -27,6 +27,8 @@ Similarly, `kops export kubecfg` will also require passing either the `--admin`
 * The `kops update cluster` command will now refuse to run on a cluster that
 has been updated by a newer version of kops unless it is given the `--allow-kops-downgrade` flag.
 
+* Kops will install the Cinder plugin and `StorageClass` on OpenStack clusters. 
```

Comment:
I do not see the logic of StorageClass in this PR?

Reply:
Yeah, this one somehow escaped the final template. Added back in.